src/content/docs/network-performance-monitoring/troubleshooting/snmp-walk.mdx

```diff
@@ -11,63 +11,75 @@ redirects:
+Run one of the following depending on your SNMP device version:
 
-```bash
-snmpwalk -v 2c -On -c $COMMUNITY $IP_ADDRESS .1.3.6.1.2.1.1.2.0
-```
-
-Where `$COMMUNITY` is your SNMP community string and `$IP_ADDRESS` is the target device IP
+<CollapserGroup>
+  <Collapser
+    id="snmp-v2c-example"
+    title="SNMP v2c example"
+  >
+  ```bash
+  snmpwalk -v 2c -On -c $COMMUNITY $IP_ADDRESS .1.3.6.1.2.1.1.2.0
+  ```
 
-**SNMP v3 Example:**
+  Where `$COMMUNITY` is your SNMP community string and `$IP_ADDRESS` is the target device IP.
+  </Collapser>
+  <Collapser
+    id="snmp-v3-example"
+    title="SNMP v3 example"
+  >
+  ```bash
+  snmpwalk -v 3 -On -l $LEVEL -u $USERNAME -a $AUTH_PROTOCOL -A $AUTH_PASSPHRASE -x $PRIV_PROTOCOL -X $PRIV_PASSPHRASE $IP_ADDRESS .1.3.6.1.2.1.1.2.0
+  ```
 
-```bash
-snmpwalk -v 3 -On -l $LEVEL -u $USERNAME -a $AUTH_PROTOCOL -A $AUTH_PASSPHRASE -x $PRIV_PROTOCOL -X $PRIV_PASSPHRASE $IP_ADDRESS .1.3.6.1.2.1.1.2.0
-```
-Replacing the following settings as necessary: 
+  Replacing the following settings as necessary: 
 
-```
-$LEVEL == noAuthNoPriv | authNoPriv | authPriv
-$USERNAME == SNMP v3 User
-$AUTH_PROTOCOL == MD5 | SHA
-$AUTH_PASSPHRASE == Authentication passphrase
-$PRIV_PROTOCOL == DES | AES
-$PRIV_PASSPHRASE == Privacy passphrase
-$IP_ADDRESS == target device IP
-```
+  ```bash
+  $LEVEL == noAuthNoPriv | authNoPriv | authPriv
+  $USERNAME == SNMP v3 User
+  $AUTH_PROTOCOL == MD5 | SHA
+  $AUTH_PASSPHRASE == Authentication passphrase
+  $PRIV_PROTOCOL == DES | AES
+  $PRIV_PASSPHRASE == Privacy passphrase
+  $IP_ADDRESS == target device IP
+  ```
 
-The expected output from this command is something similar to this in your console: 
+  The following is an example of the expected output for this command: 
 
-```
-.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.9.1.46
-```
+  ```bash
+  .1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.9.1.46
+  ```
+  </Collapser>
+</CollapserGroup>
 
-### Capturing Full SNMP Walk
+### Capturing full SNMP walk [#capturing-snmpwalk]
 
-It is often desirable to capture the output of walking every OID available on your device(s). This output is especially used when creating new SNMP profiles for `ktranslate` and is a required artifact when [opening a profile request](https://github.com/kentik/snmp-profiles/issues/new/choose) on the GitHub repo.
+You may want to capture the output of walking every OID available on your devices. This output is used when creating new SNMP profiles for `ktranslate`, and it's a requirement to [open a profile request on GitHub](https://github.com/kentik/snmp-profiles/issues/new/choose).
 
-The primary differences in your command for this are changing the target to a root `.` and redirecting the output to a file that you can gather data from later. 
+The primary differences in your command for this are changing the target to a root `.`, and redirecting the output to a file that you can gather data from later. 
 
-**SNMP v2 Example:**
+**SNMP v2 and v3 example**
```

Comment:
@x8a - this is not a v3 example; if we need to include both I can provide a sample here but my assumption was since it was such a small change the reader could look at the previous v3 example to make their own edit there

Reply:
I thought the same example works for both version but if it's not the case, let's add a specific one for v3 👌🏻.

Reply:
```bash
snmpwalk -v 3 -On -l $LEVEL -u $USERNAME -a $AUTH_PROTOCOL -A $AUTH_PASSPHRASE -x $PRIV_PROTOCOL -X $PRIV_PASSPHRASE $IP_ADDRESS . >> snmpwalk.out
```